guides/source/debugging_rails_applications.md

```diff
@@ -286,6 +286,30 @@ noticeable with large amounts of logging, but it's a good practice to employ.
 INFO: This section was written by [Jon Cairns at a StackOverflow answer](https://stackoverflow.com/questions/16546730/logging-in-rails-is-there-any-performance-hit/16546935#16546935)
 and it is licensed under [cc by-sa 4.0](https://creativecommons.org/licenses/by-sa/4.0/).
 
+SQL Query Comments
+------------------
+
+Databases can be configured to log slow queries, or instrumented with various kinds of tooling. SQL statements can be
+commented with runtime information tags to trace troublesome SQL statements back to the application code that generated
```

Comment:
This first sentence is still a bit confusing to me, or maybe I'm not fully understanding the query comments 😄 . Can slow query logs be used for query comments as well?
If so, I think we maybe want to connect these two sentences?
Maybe something like:
```suggestion
Databases can be configured to log slow queries, or instrumented with various kinds of tooling. SQL statements can be
commented with this runtime information to trace troublesome SQL statements back to the application code that generated
```

Reply:
'Runtime information tags' is controller, action, job, etc. I think adding this might clear up the confusion?

How to use these query comments: they are always added but especially useful in places like:
- your logs, if you have configured your database ([MySQL](https://dev.mysql.com/doc/refman/5.7/en/slow-query-log.html), [PostgreSQL](https://www.postgresql.org/docs/14/runtime-config-logging.html#GUC-LOG-MIN-DURATION-STATEMENT)) to log queries taking longer than a certain amount of time
- the overview of currently running queries (e.g. [MySQL](https://dev.mysql.com/doc/refman/5.7/en/show-processlist.html), [PostgreSQL](https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STAT-ACTIVITY-VIEW))
- end-to-end tracing as shown in this [Google CloudSQL blog post](https://cloud.google.com/blog/topics/developers-practitioners/introducing-sqlcommenter-open-source-orm-auto-instrumentation-library)

Reply:
Yes, I think adding that can make things even clearer. 👍

Reply:
Changed this a bit and also moved it up to directly follow the "Verbose Query Logs" section.